pkgs/stdenv/cross/default.nix

```diff
@@ -34,6 +33,12 @@ in lib.init bootStages ++ [
     allowCustomOverrides = true;
   })
 
+  # Prevent assumptions about the number of stage from creeping back in.
```

Comment:
```suggestion
  # Prevent assumptions about the number of stages from creeping back in.
```

Reply:
Not exactly sure why this works. Could you elaborate on that?

Reply:
Added further explanation in 4b09f4d2c303